docs/router/src/reference/layouts.md

```diff
@@ -0,0 +1,19 @@
+# Layouts
+
+Layouts allow you to 
+
+[`Outlet`]s tell the router where to render content in layouts. In the following example,
```

Comment:
The initial Layouts sentence is incomplete, and - really pedantic, and I apologize ahead of time lol - when reading this it may be nicer to see: 

`[Outlet] tells the router ...`

Looks like this right now: 
![image](https://github.com/DioxusLabs/dioxus/assets/12686470/a3d5be36-7972-4325-b05c-3f12ee8741f2)

Reply:
Oops, should be fixed now